src/common/dataTransaction.ts

```
@@ -573,7 +575,7 @@ export class SignedDataTransaction
     const recoveredAddress = getBytes(
       recoverAddress(prehash, hexlify(this.signature))
     );
-    return arrayCompare(recoveredAddress, this.signer);
+    return constantTimeEqual(new Uint8Array(recoveredAddress), new Uint8Array(this.signer));
```

ESLint

[error] Replace `new·Uint8Array(recoveredAddress),·new·Uint8Array(this.signer)` with `⏎······new·Uint8Array(recoveredAddress),⏎······new·Uint8Array(this.signer)⏎····`

(prettier/prettier)